package.json

```diff
@@ -186,7 +196,7 @@
 				"program": "./out/src/debug/debug_entry.js",
 				"runtime": "node",
 				"variables": {
-					"sdkPath": "dart.getSdkPath"
+					"debugSettings": "dart.getDebugSettings"
```

Comment:
@devoncarew Managed to handle this (in a way that doesn't involve manually messing with that file).. In Code 1.6 we gained the ability to edit `launch.json` programatically. I've added a function for "project upgrades" which will run automatically at activation and fix this (added to this PR).

Unfortunately the formatting sucks (https://github.com/Microsoft/vscode/issues/15062), but even if it gets fixed in 1.8, I'm not sure it's worth waiting till that releases/people upgrade before shipping.